source/tutorial/acceptance-test.md

```diff
@@ -6,7 +6,7 @@ Here's a look at what we want to build before we get started.
 
 Let's work through what we want to do on the home page. We want our application to:
 
-* Show rentals on the home page
```

Comment:
I think what's there is ok.  This list is an abstract list showing what we want the application to do.  Its not referring to the actual "rentals" route yet.